alcs-api/src/decision/application-decision/application-decision.entity.ts

```diff
@@ -71,23 +71,23 @@ export class ApplicationDecision extends Base {
 
   @AutoMap()
   @Column({ type: 'text', nullable: true })
-  decisionMakerCode?: string;
+  decisionMakerCode: string | null;
 
   @AutoMap()
   @ManyToOne(() => CeoCriterionCode, { nullable: true })
   ceoCriterion?: CeoCriterionCode;
 
   @AutoMap()
   @Column({ type: 'text', nullable: true })
-  ceoCriterionCode?: string;
+  ceoCriterionCode: string | null;
```

Comment:
the same thing about undefined, so it is intentional that this one should always be supplied or null?